go/governance/api/api.go

```diff
@@ -233,8 +237,18 @@ func (p *ChangeParametersProposal) Equals(other *ChangeParametersProposal) bool
 // PrettyPrint writes a pretty-printed representation of ChangeParametersProposal to the given
 // writer.
 func (p *ChangeParametersProposal) PrettyPrint(_ context.Context, prefix string, w io.Writer) {
+	var changes map[string]interface{}
+	if err := cbor.Unmarshal(p.Changes, &changes); err != nil {
+		fmt.Fprintf(w, "%s  <error: %s>\n", prefix, err)
```

Comment:
Not sure what is better, this
```
Change Parameters:
    <error: cbor: cannot unmarshal positive integer into Go value of type map[string]interface {}>
    <malformed: AQ==>
```
or this 
```
Change Parameters:
    cbor: cannot unmarshal positive integer into Go value of type map[string]interface {}
```

Reply:
No opinions. The `<error>` `<malformed>` is used for unparsable transactions, that's why i used it.